airflow-core/docs/empty_plugin/empty_plugin.py

```diff
@@ -19,12 +19,11 @@
 
 from __future__ import annotations
 
-from flask import Blueprint
-from flask_appbuilder import BaseView, expose
-
 from airflow.api_fastapi.auth.managers.models.resource_details import AccessView
 from airflow.plugins_manager import AirflowPlugin
 from airflow.providers.fab.www.auth import has_access_view
+from flask import Blueprint
+from flask_appbuilder import BaseView, expose
```

Comment:
This looks like ruff isort got confused/lost some config? Either way the previous order was correct.

Reply:
Yeah. I will take a look at that. It changed several times because on top of the predefined order we have a way to let ruff know what is first and 3rd party. Let me see to that.

Reply:
Good news . found and fxed the issue. the "known-first-party" setting should be added in isort not in isort.sections in pyproject.toml

Bad news (well not so bad) my PR has now slightly LESS than 3000 files. Achievement un-unlocked.